src/metabase/api/search.clj

```diff
@@ -110,7 +110,7 @@
   "Add a `WHERE` clause to the query to only return Collections the Current User has access to; join against Collection
   so we can return its `:name`."
   [honeysql-query                                :- ms/Map
-   collection-id-column                          :- keyword?
```

Comment:
was there a reason to remove the schema here?

Reply:
It's no longer a keyword, because sometimes it's `[:coalesce ...]` - but I just removed this "temporarily" and forgot to add back a new schema. Adding that back now, thanks for catching this!